### PR TITLE
Update metrics-definitions.md

### PR DESCRIPTION
**Pull request by craigwillis-mm**
Changes to the wording of Pending status.

### CODE REVIEW

**Review by emilyhollinger — DISMISSED** [review]

File: operations/business-operations/analytics/metrics-definitions.md [hunk 1]
```diff
@@ -237,8 +237,8 @@ Net Promoter Score is a standardized measure used by many organizations to measu
 #### Ticket Status
 
 * **New -** Ticket created that has not been assigned a support agent.
-* **Pending -** The support agent asks the customer a question and is waiting for their response.
-* **Pending - Do not Close -** Checkbox on the ticket. This is used for tickets that are expected to be open for a long period of time. This stops the clock. Examples are customer requests ticket remain open after a solution is provided, the customer is on vacation, or the customer is out ill.
+* **Waiting on customer -** The support agent asks the customer a question and is waiting for their response.
+* **Waiting on customer - Do not Close -** Checkbox on the ticket. This is used for tickets that are expected to be open for a long period of time. This stops the clock. Examples are customer requests ticket remain open after a solution is provided, the customer is on vacation, or the customer is out ill.
 * **On Hold -** The support agent reaches out to an internal team and is waiting to hear back. Any ticket that is tied to a JIRA ticket is placed on hold. Internal teams include product, development or customer success.
```
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
* **On Hold -** The support agent reaches out to an internal team and is waiting to hear back. Internal teams include product, development, or customer success. Any ticket that is tied to a Jira ticket is placed "On Hold".
```

File: operations/business-operations/analytics/metrics-definitions.md [hunk 2]
```diff
@@ -237,8 +237,8 @@ Net Promoter Score is a standardized measure used by many organizations to measu
...
 * **Solved -** Support agent provides a solution to the customer. Status in the solved status for 48 hours before it is set to “Closed” status While the ticket is in a “Solved” status any response from the customer will re-open the ticket.
```
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
* **Solved -** Support agent provides a solution to the customer. A ticket remains in a "Solved" status for 48 hours before it is set to “Closed”. While the ticket is in a “Solved” status any response from the customer will reopen the ticket.
```

File: operations/business-operations/analytics/metrics-definitions.md [hunk 3]
```diff
@@ -237,8 +237,8 @@ Net Promoter Score is a standardized measure used by many organizations to measu
...
 * **Closed -** If no response is recorded a ticket in a Solved status will be set to Closed automatically after 48 hours. A response to the ticket will open a new ticket.
```
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
* **Closed -** If no response is recorded after 48 hours, a ticket in a "Solved" status will automatically be set to "Closed". Any response to the ticket after that period of time will open a new ticket.
```

File: operations/business-operations/analytics/metrics-definitions.md [hunk 4]
```diff
@@ -237,8 +237,8 @@ Net Promoter Score is a standardized measure used by many organizations to measu
 #### Ticket Status
 
 * **New -** Ticket created that has not been assigned a support agent.
-* **Pending -** The support agent asks the customer a question and is waiting for their response.
-* **Pending - Do not Close -** Checkbox on the ticket. This is used for tickets that are expected to be open for a long period of time. This stops the clock. Examples are customer requests ticket remain open after a solution is provided, the customer is on vacation, or the customer is out ill.
+* **Waiting on customer -** The support agent asks the customer a question and is waiting for their response.
+* **Waiting on customer - Do not Close -** Checkbox on the ticket. This is used for tickets that are expected to be open for a long period of time. This stops the clock. Examples are customer requests ticket remain open after a solution is provided, the customer is on vacation, or the customer is out ill.
```
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
* **Waiting on customer - Do not Close -** Checkbox on the ticket. This is used for tickets that are expected to be open for a long period of time. This stops the clock. 
Examples include: The customer requests that the ticket remain open after a solution is provided, the customer is on vacation, or the customer is out ill.
```

**Review by justinegeffen — CHANGES_REQUESTED** [review]
Thanks, @craigwillis-mm. I've made some suggestions around the wording. Let me know if anything is unclear. :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**